inc/utility/namespace.php

```diff
@@ -1279,7 +1279,7 @@ function rmrdir( $dirname, $only_empty = false ) {
 
 
 /**
- * Comma separated, Oxford comma, localized and between the last two items
+ * Comma separated, Oxford semicolon localized and between the last two items
```

Comment:
Comment should read: 'Semicolon separated list, with a localized 'and' between the last two items'